docs/best_practices.md

```diff
@@ -0,0 +1,31 @@
+# Best Practices
```

Comment:
These are new docs

Reply:
love the idea of a best practices section